frontend/src/queries/query.ts

```diff
@@ -133,15 +133,15 @@ async function executeQuery<N extends DataNode>(
 }
 
 // Return data for a given query
-export async function query<N extends DataNode>(
+export async function performQuery<N extends DataNode>(
```

Comment:
Now we have `performQuery` right next to `executeQuery`...

Are there other naming options? 😆

Reply:
Those two should probably be one function -> guess that's for a follow up